lib/class/api.class.php

```diff
@@ -3,7 +3,7 @@
 /**
  *
  * LICENSE: GNU Affero General Public License, version 3 (AGPLv3)
- * Copyright 2001 - 2015 Ampache.org
+ * Copyright 2001 - 2016 Ampache.org
```

Comment:
I think this change is more general than these files. Just commenting to remember to do it.